tests/unit_tests/forward_model_runner/test_job.py

```diff
@@ -67,14 +68,50 @@ def max_memory_per_subprocess_layer(layers: int) -> int:
         max_seen = 0
         for status in job.run():
             if isinstance(status, Running):
-                max_seen = max(max_seen, status.max_memory_usage)
+                max_seen = max(max_seen, status.runstate.max_rss)
         return max_seen
 
     max_seens = [max_memory_per_subprocess_layer(layers) for layers in range(3)]
     assert max_seens[0] < max_seens[1]
     assert max_seens[1] < max_seens[2]
 
 
+@pytest.mark.usefixtures("use_tmpdir")
+def test_memory_profile_in_running_events():
+    scriptname = "increasing_memory.py"
+    with open(scriptname, "w", encoding="utf-8") as script:
+        script.write(
+            """#!/usr/bin/env python
+import time
+somelist = []
+
+for _ in range(20):
+    # 1 Mb allocated pr iteration
+    somelist.append(b' ' * 1024 * 1024)
+    time.sleep(0.02)
+"""
+        )
```

Comment:
Can we format this with textwrap dedent instead?

Reply:
fixed